.github/workflows/cibuildsdist.yml

```diff
@@ -0,0 +1,65 @@
+name: Build CI sdist and wheel
+
+on:
+  # Save CI by only running this on release branches or tags.
+  push:
+    branches:
+      - main
+      - v[0-9]+.[0-9]+.x
+    tags:
+      - v*
+  # Also allow running this action on PRs if requested by applying the
+  # "Run cibuildsdist" label.
```

Comment:
Not sure we really need a separate label for this one.

Reply:
I would use the same label we use for the cibuild wheel tests?

Reply:
Wasn't sure, but have modified it to use the same label.